gruntfile.js

```diff
@@ -803,6 +803,13 @@ module.exports = function (grunt) {
                         { src: "./tools/config/config.json", dest: `./tools/config/browser/es5/ai.config${configVer}.cfg.json` },
                         { src: "./tools/config/config.json", dest: `./tools/config/browser/es5/ai.config${configMajorVer}.cfg.json` }
                     ]
+                },
+
+                testConfig: {
+                    files: [
+                        { src: "./tools/config/config.json", dest: `./tools/config/browser/es5/ai_test.config${configVer}.cfg.json` },
```

Comment:
Do we "really" want to copy the single "default" one to the test, I was suggesting that we actually have 2 checked in, so this would copy the test one as the test

Reply:
Thanks Nev for pointing it out! I just found it as well and it should be "test-config", I already updated the PR!